docs/farming.md

```diff
@@ -48,10 +48,9 @@ If you're connected directly without any router, then again nothing needs to be
 --execution wasm `
 --blocks-pruning archive `
 --state-pruning archive `
---dsn-disable-private-ips `
```

Comment:
@ImmaZoni could you ensure these changes get updated into the pre-release version of the main docs please?